src/MatrixFields/field_matrix_solver.jl

```diff
@@ -66,6 +66,9 @@ function FieldMatrixSolver(
     return FieldMatrixSolver(alg, cache)
 end
 
+Base.zero(solver::FieldMatrixSolver) =
+    FieldMatrixSolver(zero(solver.alg), zero(solver.cache))
```

Comment:
This method is unnecessary. The algorithm is purely symbolic, and the cache does not need to be initialized.